v2/devices/enchilada.yml

```diff
@@ -0,0 +1,121 @@
+name: "OnePlus 6/6T"
+codename: "enchilada"
+formfactor: "phone"
+aliases: []
+doppelgangers: []
```

Comment:
Do fajita need a different config/boot image?

Reply:
All is same.